CHANGELOG.md

```diff
@@ -3,6 +3,61 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## Windows/Linux/macOS 5.2.0 - 2017-02-14
+Updated PECL release packages. Here is the list of updates:
+
+### Added
+- Added support for Always Encrypted with basic CRUD functionalities (see [here](https://github.com/Microsoft/msphpsql/wiki/Features#aebindparam))
+  - Support for Windows Certificate Store (use connection keyword ColumnEncryption)
+  - Support for inserting into and modifying an encrypted column
+  - Support for fetching from an encrypted column
+- Added support for PHP 7.2
+- Added support for MSODBC 17
```

Comment:
can we be a bit more consistent with the naming? Sometimes we say "MSODBC" and sometimes "ODBC Driver". I think it should be "MS ODBC Driver", what do you think?

Reply:
sounds good